readthedocs/doc_builder/exceptions.py

```diff
@@ -36,6 +36,9 @@ class BuildUserError(BuildBaseException):
         "To resolve this error, double check your project configuration and installed "
         "dependencies are correct and have not changed recently."
     )
+    BUILD_COMMANDS_WITHOUT_OUTPUT = gettext_noop(
+        'No "output/" folder detected on the root of the repository after building the project.'
```

Comment:
I don't love this error message (`root of the repository` seems hard to parse), but I don't have a great suggestion for improving it..

Maybe:

```suggestion
        'No "output/" folder was created in the root directory after running `build.commands`.'
```

Reply:
We could say "checkout path", but even that's probably not clear. "root directory" implies `/` to me though.

We could not mention the path in the error, but instead do a better job of highlighting requirements in the documentation. If there was something obvious to reference in our docs, that feels most helpful to the user. Nomenclature here will be difficult to pin down.